csv/src/main/java/org/apache/metamodel/csv/CsvConfiguration.java

```diff
@@ -195,4 +199,17 @@ public String toString() {
                 + ", separatorChar=" + separatorChar + ", quoteChar=" + quoteChar + ", escapeChar=" + escapeChar
                 + ", failOnInconsistentRowLength=" + failOnInconsistentRowLength + "]";
     }
+    
+    public ICSVParser createParser() {
```

Comment:
Hmm I'd ideally like this somewhere else than in CsvConfiguration. I consider CsvConfiguration part of MetaModel's public API and so it would be bad to also expose OpenCSV classes. Just in case we ever need to switch to another CSV parser library.